htmx-spring-boot/src/main/java/io/github/wimdeblauwe/htmx/spring/boot/mvc/HtmxView.java

```diff
@@ -17,7 +17,9 @@
  * this class.
  *
  * @since 3.6.0
+ * @deprecated since 4.0.0 for removal in 4.1.0 in favor of <a href="https://docs.spring.io/spring-framework/reference/web/webmvc-view/mvc-fragments.html">HTML Fragments</a> support.
```

Comment:
Why should we keep this in 4.0.0? Can't we just get rid of it all together since we are doing other breaking changes anyway?

Reply:
There's no strong reason to keep it for 4.0.0, but I thought it made more sense to mark it as deprecated before we completely remove it from our codebase.

Reply:
ok.